src/auth/auth-settings.c

```diff
@@ -261,6 +261,7 @@ static const struct setting_define auth_setting_defines[] = {
 	DEF(SET_STR, verbose_passwords),
 	DEF(SET_BOOL, ssl_require_client_cert),
 	DEF(SET_BOOL, ssl_username_from_cert),
+	DEF(SET_BOOL, auth_external_ignore_authzid),
```

Comment:
Otherwise ok, but "auth_" prefix is automatically added to all these settings. So you've now actually created auth_auth_external_ignore_authzid setting.